bin/jsii-release-golang

```diff
@@ -122,10 +122,19 @@ pushd ${target_repo_dir}
 git config user.name ${GIT_USER_NAME}
 git config user.email ${GIT_USER_EMAIL}
 
-# -B to create if it doesn't exist, reset otherwise
-git checkout -B ${GIT_BRANCH}
```

Comment:
This didn't do what I thought it did, and always created a new branch even if `git checkout` would work.